### PR TITLE
add drawing app v2

### PR DESCRIPTION
**Pull request by tigercoding56**
now that i fixed some of the cross-branch polution (i think i got all of it ) 
lets resubmit a PR
DrawApp
same as bevore

### CODE REVIEW

**Comment by fgaz** [comment]
I think either is fine but real name is preferred

File: docs/apps.rst [hunk 1]
```diff
@@ -51,6 +51,8 @@ Applications
 
 .. automodule:: apps.demo
 
+.. automodule:: draw
```
Comment (fgaz) [inline docs/apps.rst:54]: `Draw` with a capital D. rename `apps/draw.py` to `apps/Draw.py` to match

Comment (tigercoding56) [reply docs/apps.rst]: ok

**Comment by tigercoding56** [comment]
> I think either is fine but real name is preferred

ok

**Comment by fgaz** [comment]
There is another error:

> Screenshots must be included in the full module documentation

Like this:

https://github.com/wasp-os/wasp-os/blob/99272f482fb6eadd26afe42a90c06d0db4a240dd/apps/Level.py#L11-L12

By the way, you can also run those tests locally with `make check`, without having to wait for me to approve the CI run

**Comment by tigercoding56** [comment]
> .. figure:: res/LevelApp.png

ok , thank you 
(i totally missed that)

**Comment by tigercoding56** [comment]
> By the way, you can also run those tests locally with `make check`, without having to wait for me to approve the CI run

im so stupid how did i miss that

**Comment by tigercoding56** [comment]
what does  "Failed to import ecdsa, cannot do signing" mean?

**Comment by fgaz** [comment]
> what does "Failed to import ecdsa, cannot do signing" mean?

I don't know, but that's not a fatal error. What's making the build fail is a bit up in the log:

https://github.com/wasp-os/wasp-os/actions/runs/4310693997/jobs/7533224709#step:8:128
```
build finished with problems, 1 warning.
[...]
make[1]: *** [Makefile:27: html] Error 1
make[1]: Leaving directory '/home/runner/work/wasp-os/wasp-os/docs'
make: *** [Makefile:103: docs] Error 2
make: *** Waiting for unfinished jobs....
```

And if you look further up for that warning:

https://github.com/wasp-os/wasp-os/actions/runs/4310693997/jobs/7533224709#step:8:53
```
/home/runner/work/wasp-os/wasp-os/apps/Draw.py:docstring of Draw:5: WARNING: Unexpected indentation.
```

**Comment by tigercoding56** [comment]
> > what does "Failed to import ecdsa, cannot do signing" mean?
> 
> I don't know, but that's not a fatal error. What's making the build fail is a bit up in the log:
> 
> https://github.com/wasp-os/wasp-os/actions/runs/4310693997/jobs/7533224709#step:8:128
> 
> ```
> build finished with problems, 1 warning.
> [...]
> make[1]: *** [Makefile:27: html] Error 1
> make[1]: Leaving directory '/home/runner/work/wasp-os/wasp-os/docs'
> make: *** [Makefile:103: docs] Error 2
> make: *** Waiting for unfinished jobs....
> ```
> 
> And if you look further up for that warning:
> 
> https://github.com/wasp-os/wasp-os/actions/runs/4310693997/jobs/7533224709#step:8:53
> 
> ```
> /home/runner/work/wasp-os/wasp-os/apps/Draw.py:docstring of Draw:5: WARNING: Unexpected indentation.
> ```
Oh  , i am sorry for being that stupid

**Comment by beardeddude** [comment]
As you may know after the merge of #403 the naming conventions and file placement have changed slightly. See https://wasp-os.readthedocs.io/en/latest/appguide.html#app-naming-conventions-and-placement for details.

**Comment by tigercoding56** [comment]
> As you may know after the merge of #403 the naming conventions and file placement have changed slightly. See https://wasp-os.readthedocs.io/en/latest/appguide.html#app-naming-conventions-and-placement for details.

ok

**Comment by tigercoding56** [comment]
I'll close this as school started again and i do not have as much time as i had during break if anyone wants to use this application on their own versions of wasp-OS go ahead

**Pull request closed without merging**